src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

```diff
@@ -78,17 +78,17 @@ Here's an overview of the capabilities of each user type:
   </tbody>
 </table>
 
-Remember that the user type governs a user's theoretical access. In practice, users can have [roles assigned](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts) that restrict their access further.
+Remember that the user type overrides any [role-related permissions](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#user-type-groups-relation).
 
 Want to understand how billing works? See [User billing calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing).
 
 ## Detailed user type comparison table [#user-type-comparison-table]
 
 Here's a detailed comparison of capabilities by user type. 
 
-Important points about this table: 
-* User type is how you control user access based on a user's long term responsibilities. The more granular way to control access is using roles. [Learn more about how user access is controlled.](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#user-type-groups-relation)
-* Many of the features in the table refer to access to the UI experiences, not to the underlying data itself. All users, including basic users, can query all data in the accounts they have access to, and can create and view custom charts. Put another way: basic users can access APM data and browser monitoring data and more, but don't have access to those curated UI experiences. 
+Important points about this table:
+* The user type is meant to be a long-term setting. User-type-related restrictions override role-based permissions. [Learn more about user access factors.](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#user-type-groups-relation)
```

Comment:
Consider elaborating on what you mean by a long-term setting, perhaps with an example.